enterprise/backend/src/metabase_enterprise/data_editing/api.clj

```diff
@@ -98,10 +95,13 @@
   (let [rows (apply-coercions table-id rows)
         res  (perform-bulk-action! :bulk/create table-id rows)]
     (doseq [row (:created-rows res)]
-      (events/publish-event! :event/data-editing-row-create
-                             {:table_id    table-id
-                              :created_row row
-                              :actor_id    api/*current-user-id*}))
+      (events/publish-event! :event/action.success
+                             {:action        :row/create
+                              :invocation_id (nano-id/nano-id)
+                              :actor_id      api/*current-user-id*
+                              :table_id      table-id
+                              :result        {:created_row row
+                                              :table-id    table-id}}))
```

Comment:
why do you need table_id at both top level and inside result?

Reply:
I'm keeping it at the top level so that it populates the column that you use for optimizing subscriptions.

I also keep it in the result so that it provides a logical bundle, this is where it truly belongs in my mind, the fact that it's also in the "envelope" should be an invisible implementation detail.